schedule/yam/agama/agama_encrypted_lvm.yaml

```diff
@@ -3,10 +3,12 @@ name: agama_dolomite_encrypted_lvm
 description: >
   Playwright test on agama dolomite for installation with fully encrypted disk,
   with LVM in use.
+vars:
+    AGAMA_TEST: 'tests/encrypted_lvm.spec.ts'
```

Comment:
all our test to run from openQA will be in `test/` so we can omit pass that, we can just hardcode it in the call.